AbletonMCP_Remote_Script/handlers/session.py

```
@@ -0,0 +1,1118 @@
+        return result
+    except Exception as e:
+        if ctrl:
+            ctrl.log_message("Error getting session info: " + str(e))
+        raise
+
+
+def set_tempo(song, tempo, ctrl=None):
+    """Set the tempo of the session."""
+    try:
+        song.tempo = tempo
+        return {"tempo": song.tempo}
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)